sites-config/ci_config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
oh was looking at the sites starter for unrelated studio stuff and noticed this file can just be ci.json now?
https://github.com/yext/pages-starter-react-locations/blob/main/sites-config/ci.json

Reply:
cool, I will update!